dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -905,7 +706,7 @@ data:
                         "refId": "A"
                     }
                 ],
-                "title": "Pipeline Scheduling SLO",
+                "title": "Pipeline Scheduling Overhead (not an SLO or alertable metric)",
```

Comment:
Maybe we should start a new pipeline specific dashboard (after this batch of changes) and provide a link to it for more details. Wdyt? This dashboard is supposed to be just for SLOs and a new dashboard would be easier to modify if only your team will touch it.

Reply:
yeah I'm going to change it to `(alert remove, panel remove occurring soon)` and then will craft the PR to remove after this merges

but if you come up with something else before I push, I'll try to catch it and adjust @amisstea